cpu/nrf52/Makefile.features

```diff
@@ -12,6 +12,12 @@ ifneq (,$(filter nrf52811xxaa nrf52820xxaa nrf52833xxaa nrf52840xxaa,$(CPU_MODEL
   FEATURES_PROVIDED += netif_openwsn
 endif
 
+# The nrf usb stack is supported by tinyUSB.
```

Comment:
```suggestion
# tinyUSB stack is supported on nRF52 w/ USB peripheral.
```

'nrf usb stack' is rather obscur to me as there is no such thing.
You may squash directly.

Reply:
Thanks, good point.